source/jormungandr/tests/proxy_realtime_integration_tests.py

```diff
@@ -0,0 +1,224 @@
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+from collections import namedtuple
+
+from .tests_mechanism import AbstractTestFixture, dataset
+from jormungandr.realtime_schedule import realtime_proxy, realtime_proxy_manager
+from jormungandr.schedule import RealTimePassage
+import datetime
+from nose.tools import eq_
+import pytz
+from .check_utils import is_valid_stop_date_time, get_not_null
+from mock import MagicMock
+from navitiacommon import type_pb2
+
+
+MOCKED_PROXY_CONF = ('[{ "destination_code_type": "KisioDigital",'
+                     '"object_id_tag": "KisioDigital",'
+                     '"id": "KisioDigital",'
+                     '"class": "jormungandr.realtime_schedule.timeo.Timeo",'
+                     '"args": {'
+                     '"timezone": "Europe/Paris",'
+                     '"service_url": "http://XXXX",'
+                     '"timeout": 15,'
+                     '"service_args": {'
+                     '"serviceID": "X",'
+                     '"EntityID": "XX",'
+                     '"Media": "XXX"}}}]')
+
+
+class Obj(object):
+    pass
+
+@dataset({"basic_schedule_test": {"proxy_conf": MOCKED_PROXY_CONF}})
+class TestDepartures(AbstractTestFixture):
+    query_template = 'stop_points/{sp}/stop_schedules?from_datetime={dt}&show_codes=true{data_freshness}'
+
+    def test_stop_schedule_without_rt(self):
+        query = self.query_template.format(sp='C:S0', dt='20160102T1100', data_freshness='')
+        from jormungandr import i_manager
+        instance = i_manager.instances.get('basic_schedule_test')
+        rt_system = instance.realtime_proxy_manager.get('KisioDigital')
+
+        assert rt_system
+        resp = Obj()
+        resp.status_code = 200
+        resp.json = MagicMock(return_value={})
+        rt_system._call_timeo = MagicMock(return_value=resp)
+        rt_system._make_url = MagicMock(return_value={"http://kisiodigital.com"})
+        response = self.query_region(query)
+        stop_schedules = response['stop_schedules']
+        assert len(stop_schedules) == 1
+        stop_times = stop_schedules[0]['date_times']
+        assert len(stop_times) == 1
+        stop_time = stop_times[0]
+        assert stop_time['data_freshness'] == 'base_schedule'
+        assert stop_time['date_time'] == '20160102T113000'
+
+    def test_stop_schedule_with_rt_and_without_destination(self):
+        query = self.query_template.format(sp='C:S0', dt='20160102T1100', data_freshness='')
+        from jormungandr import i_manager
+        instance = i_manager.instances.get('basic_schedule_test')
+        rt_system = instance.realtime_proxy_manager.get('KisioDigital')
+
+        assert rt_system
+        resp = Obj()
```

Comment:
I think you should do something more like timeo_test.py:252 or bss_provider_test.py:51